### PR TITLE
#226 - meetup activities

### LINKED ISSUE #226: [Dashobard] Meetup activities
We should implement CRUD for meetup activities on our website. This resource should store things related to meetings at which someone from our company presented something - meetups like Laravel Poland Meetup, PHPers etc.

Meetup activity should have fields like that:
- photo,
- title_of_presentation,
- presenter_name,
- text/description,
- created_at/published_at,
- associate links.

Meetup activity should be created for two languages - `pl` and `en`.

### PR DESCRIPTION
**Pull request by kamilpiech97**
Should close #226.
Added meetup activity model and CRUD views in admin panel. Some of meetup activity fields are translatable via [laravel-translatable](https://spatie.be/docs/laravel-translatable/v6/introduction).

Preview:

image

### CODE REVIEW

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. Why this title is pascalCase? I believe it should be "Aktywność na meetupach"

image


2. This validation message is a bit unclear for me

image


3. When I don't enter presenter and I want to save activity, I see error

image

4. Maybe in field "Powiązany link" we should give some placeholder (or initialize this field with "https://" )  because now user  when enters URL like "www.example.com" he sees, that is incorrect URL, because URL should start with http:// or https:// 


On Slack for example it looks like this:

image

**Comment by kamilpiech97** [comment]
> 1. Why this title is pascalCase? I believe it should be "Aktywność na meetupach"
> 
> image
> 
> 2. This validation message is a bit unclear for me
> 
> image
> 
> 3. When I don't enter presenter and I want to save activity, I see error
> 
> image
> 
> 4. Maybe in field "Powiązany link" we should give some placeholder (or initialize this field with "https://" )  because now user  when enters URL like "www.example.com" he sees, that is incorrect URL, because URL should start with http:// or https://
> 
> On Slack for example it looks like this:
> 
> image

2,3,4 fixed
1 PascalCase is on all pages for all labels/titles, I would like to change that but I must find proper way. Now, I would like to merge that into main.

**Comment by krzysztofrewak** [comment]
@kamilpiech97, look how I changed labels in #246

**Comment by kamilpiech97** [comment]
> @kamilpiech97, look how I changed labels in #246

thanks.
@EwelinaSkrzypacz number **one** fixed :D

File: app/Filament/Resources/CaseStudyResource.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 class CaseStudyResource extends Resource
 {
     protected static ?string $model = CaseStudy::class;
-    protected static ?string $label = "case study";
+    protected static ?string $label = "Case study";
```
Comment (krzysztofrewak) [inline app/Filament/Resources/CaseStudyResource.php:22]: Why is that?

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: app/Filament/Resources/CaseStudyResource.php (hunk 1 above)
Comment (krzysztofrewak) [reply app/Filament/Resources/CaseStudyResource.php:22]: Previous one was better: 
obraz

Comment (kamilpiech97) [reply app/Filament/Resources/CaseStudyResource.php]: fixed and adjusted in other resources

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Comment by krzysztofrewak** [comment]
What about situations like last week's conference where you and Karol presented something together?

**Comment by kamilpiech97** [comment]
> What about situations like last week's conference where you and Karol presented something together?

Should be added like this:
image

**Pull request merged**